File: Excel/400-499/423/423 Challenge.R
library(tidyverse)
library(readxl)

input = read_excel("Excel/423 Split Case Sensitive Alphabets and Numbers.xlsx", range = "A1:A10")
test  = read_excel("Excel/423 Split Case Sensitive Alphabets and Numbers.xlsx", range = "B1:B10")

pattern = ("[A-Z]+|[a-z]+|[0-9]+")

result = input %>%
  mutate(splitted = map_chr(Data, ~str_extract_all(., pattern) %>% unlist() %>% 
                              str_c(collapse = ", "))) 

identical(result$splitted, test$`Expected Answer`)
# [1] TRUE

File: Excel/400-499/423/423 Challenge.py
import pandas as pd
import re

input = pd.read_excel("423 Split Case Sensitive Alphabets and Numbers.xlsx", usecols="A", nrows=10)
test = pd.read_excel("423 Split Case Sensitive Alphabets and Numbers.xlsx", usecols="B", nrows=10)

pattern = r"[A-Z]+|[a-z]+|[0-9]+"

result = input["Data"].apply(lambda x: ", ".join(re.findall(pattern, str(x))))

print(test["Expected Answer"].equals(result)) # True